app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, foreign_key: 'post_id', counter_cache: true
+
+  after_create do
+    post.increment!(:comments_count)
+  end
```

Comment:
- You can use the `after_save` Active Callback to call the method that updates the post counter, which should be a private method.